game/src/edit/bulk.rs

```diff
@@ -172,11 +172,10 @@ impl State for BulkEdit {
                     let speed = self.composite.dropdown_value("speed limit");
                     let mut edits = app.primary.map.get_edits().clone();
                     for r in &self.roads {
-                        edits.commands.push(EditCmd::ChangeSpeedLimit {
-                            id: *r,
-                            new: speed,
-                            old: app.primary.map.get_r(*r).speed_limit,
-                        });
+                        let old = app.primary.map.get_r_edit(*r);
```

Comment:
This pattern happens a few times. Think it's worth it to do something like `map.make_r_edit_cmd(*r, |change_r| change_r.speed_limit = speed;)`?

Reply:
Seems reasonable.